src/lib.rs

```diff
@@ -659,6 +735,26 @@ macro_rules! create_app {
                             .conflicts_with("enclave-id"),
                     ),
             )
+            .subcommand(
+                SubCommand::with_name("pcr")
+                    .about("Return the PCR hash value of the given input.")
+                    .arg(
+                        Arg::with_name("signing-certificate")
+                            .long("signing-certificate")
+                            .takes_value(true)
+                            .help("Takes the path to the '.pem' signing certificate and returns PCR8. Can be used to identify the certificate used to sign an EIF")
+                            .required_unless("input")
+                            .conflicts_with("input"),
+                    )
+                    .arg(
+                        Arg::with_name("input")
+                            .long("input")
+                            .takes_value(true)
+                            .help("Given a path to a file, returns the PCR hash of the bytes it contains.")
```

Comment:
Either put a `.` everywhere, either you don't put it at all

```suggestion
                            .help("Given a path to a file, returns the PCR hash of the bytes it contains")
```